### ISSUE TITLE
Out of memory

### ISSUE THREAD

**Issue by Eliepse** [Eliepse/mailgrate#3]
When updating an heavy account (> 400 mailboxes, >22 000 mails).

```bash
mmap() failed: [12] Cannot allocate memory
mmap() failed: [12] Cannot allocate memory
PHP Fatal error:  Out of memory (allocated 1426063360) (tried to allocate 4096 bytes) in /home/eliepse/mailgrate/vendor/illuminate/database/Eloquent/Model.php on line 392
mmap() failed: [12] Cannot allocate memory
PHP Fatal error:  Out of memory (allocated 1426063360) (tried to allocate 65536 bytes) in /home/eliepse/mailgrate/vendor/filp/whoops/src/Whoops/Exception/Frame.php on line 159
```

**Comment by Eliepse** [comment]
Might be interesting to pluck results, or to regularly save the new models, then flush them.

**Comment by Eliepse** [comment]
The bug was caused by excessive use of automatic leazy loading (`$with` property on models).
Fixed in commit 3b58803.
